README.md

```diff
@@ -70,14 +71,29 @@ Specify the upstream archive-data provider in your ``ape-config.yaml``:
 
 ```yaml
 ganache:
-  mainnet_fork:
-    upstream_provider: infura
+  fork:
+    ethereum:
+      mainnet:
+        upstream_provider: infura
```

Comment:
Is Infura the only provider that works with ganache, or is this just a suggestion?

Reply:
Suggestion